### ISSUE TITLE
Add the ability to pass a uv_handle_t to a new event loop

### ISSUE THREAD

**Issue by kellabyte** [joyent/libuv#1301]
It would be really great to avoid having to implement the complexity of https://github.com/joyent/libuv/blob/master/test/benchmark-multi-accept.c and avoid sending over IPC to load balance on other event loops. I find this code hard to follow and it's the part I struggle with the most out of all my libuv calling code.

Potentially latency can be improved if this isn't done over IPC?

**Comment by piscisaureus** [comment]
I agree with the general observation that this could be done better. 

The way to go IMO would be to add an easier API to share handles within a process. Then the user could create a separate thread w/ an event loop that dispatches incoming connections to the different worker threads/loops.

**Comment by kellabyte** [comment]
@piscisaureus That sounds great! :)

**Comment by indutny** [comment]
Yeah, this sounds reasonable indeed.
